lms/djangoapps/grades/tasks.py

```diff
@@ -33,6 +33,7 @@
 log = getLogger(__name__)
 
 KNOWN_RETRY_ERRORS = (DatabaseError, ValidationError)  # Errors we expect occasionally, should be resolved on retry
+RECALCULATE_GRADE_DELAY = 2  # in seconds, to prevent excessive _has_db_updated failures. See TNL-6424.
```

Comment:
By putting it in lms/envs, we can change the value without redeploying edx-platform.
However, it's unclear whether we would ever need to change the value once deployed.  Consider getting input from devops on this question.